share/completions/cryptsetup.fish

```diff
@@ -1,4 +1,7 @@
 #cryptsetup 2.3.5
+function __fish_dm_names
```

Comment:
technically this should be namespaced according to the file name, so `__fish_cryptsetup_dm_names`.
Could also just inline this function, no?